### PR TITLE
Add release notes from 0.9.0 to 0.10.2

### PR DESCRIPTION
**Pull request by savitaashture**
Add release notes from 0.9.0 to 0.10.2 as part of Jira https://issues.redhat.com/browse/SRVKP-1226 

/cc @khrm @vdemeester

### CODE REVIEW

File: openshift/release/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+## Features :sparkles:
+
+* Add timestamps in logs ([#802](https://github.com/tektoncd/triggers/pull/802))
+  
+  Add timestamp in the logs of the pipelines controller, webhook, and eventlisteners.
+
+* Improve the error message when URL form encoding is received ([#806](https://github.com/tektoncd/triggers/pull/806))
+  
+  The GitHub interceptor now detects form-encoded hook requests and returns an explicit error for this case rather than the generic JSON parsing error.
+
```
Comment (savitaashture) [inline openshift/release/release-notes.md:37]: Shall we remove these?

File: openshift/release/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+                    cpu: "500m"
+```  
+* Use Listers to fetch data in the Sink([#821](https://github.com/tektoncd/triggers/pull/821))
+  
+  EventListener ServiceAccounts now need to have "list" and "watch" verbs in addition to "get" for all triggers resources. See examples at https://github.com/tektoncd/triggers//tree/v0.10.0//examples/role-resources/triggerbinding-roles/role.yaml and https://github.com/tektoncd/triggers/tree/v0.10.0//examples/role-resources/clustertriggerbinding-roles/clusterrole.yaml
+
+* Add Timeout for EventListener Server ([#747](https://github.com/tektoncd/triggers/pull/747))
+  
+  Add Read, Write, and Idle timeout for connections to the EventListener.
+
```
Comment (savitaashture) [inline openshift/release/release-notes.md:71]: these also

File: openshift/release/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+  action required: If you were relying on the escaping, you can retain the old behaviour by adding an annotation to an affected TriggerTemplate, `triggers.tekton.dev/old-escape-quotes: "true"`
+
+## Fixes :bug:
+* Don't log Fatal when handling events in the eventlistener ([#810](https://github.com/tektoncd/triggers/pull/810))
+* Fix apiVersion for test triggerbinding ([#826](https://github.com/tektoncd/triggers/pull/826))
+* Change the webhook name to triggers-webhook ([#829](https://github.com/tektoncd/triggers/pull/829))
+ 
+  Fix an issue that caused the webhook, under certain conditions, to fail to acquire a lease and not function correctly as a result.
+* Fix EL Container Arguments in Reconciler ([#838](https://github.com/tektoncd/triggers/pull/838))
```
Comment (savitaashture) [inline openshift/release/release-notes.md:141]: How about removing entire `Fixes`?

File: openshift/release/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+
+## Misc :hammer:
+* Add multi-arch builds for triggers ([#812](https://github.com/tektoncd/triggers/pull/812))
+* Fix GitHub and GitLab capitalization ([#811](https://github.com/tektoncd/triggers/pull/811))
+* Update Tekton Pipeline Dependency Version to v0.18.0 ([#825](https://github.com/tektoncd/triggers/pull/825))
+* Separate SAs for controller/webhook deployment to allow for different permission sets ([#818](https://github.com/tektoncd/triggers/pull/818))
+* Switch webhook apiVersions to v1 ([#830](https://github.com/tektoncd/triggers/pull/830))
+* Use zaptest logger in tests ([#833](https://github.com/tektoncd/triggers/pull/833))
```
Comment (savitaashture) [inline openshift/release/release-notes.md:149]: How about removing entire `Misc` ?

File: openshift/release/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+
+## Docs :book:
+* Fixing broken link to pipeline repo ([#814](https://github.com/tektoncd/triggers/pull/814))
+* Update README for trigger selector based examples ([#839](https://github.com/tektoncd/triggers/pull/839))
+* Update tls example yaml ([#834](https://github.com/tektoncd/triggers/pull/834))
```
Comment (savitaashture) [inline openshift/release/release-notes.md:154]: this as well

File: openshift/release/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+  
+## Deprecation Notices :rotating_light:
+* Use template.ref instead of template.name ([#787](https://github.com/tektoncd/triggers/pull/787))
+
+  The template.name field is deprecated (in favor of template.ref) and will be removed in a future release.
+
+* Adds support for name/value embedded Bindings ([#768](https://github.com/tektoncd/triggers/pull/768))
+
+  The old syntax for embedded TriggerBindings (using `spec.params`) is deprecated. Use the new name/value fields instead:  
+
```
Comment (savitaashture) [inline openshift/release/release-notes.md:254]: we can remove from here as its handled in `Feature`

File: openshift/release/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+* Apply triggertemplate param default value if triggerbinding param missing from body/header ([#761](https://github.com/tektoncd/triggers/pull/761))
+
+  Triggers now applies the default value of a trigger template param if it's value cannot be resolved from a TriggerBinding. This is a BREAKING CHANGE. Previously, Triggers would throw an error saying that the binding value could not be resolved (See #568)
+
+* Set runAsUser uid for nonroot distroless image ([#789](https://github.com/tektoncd/triggers/pull/789))
+
+  Fix inconsistent uid for the controller and webhook deployment, resulting in failure of installing tekton pipeline on minikube (and other platforms.)
+
+* Fix Binding test builder to only use ref ([#769](https://github.com/tektoncd/triggers/pull/769))
+
```
Comment (savitaashture) [inline openshift/release/release-notes.md:285]: these info also can we remove?

File: openshift/release/release-notes.md [hunk 8]
```diff
@@ -0,0 +1,299 @@
...
+* remove interceptor cross namespace secret references ([#748](https://github.com/tektoncd/triggers/pull/748))
+
+  BREAKING CHANGE: any interceptors that attempt to reference a Secret outside of the EventListener's namespace will have to change to have those Secrets reside in the EventListener's namespace.
+
+* Add Clusterrole Aggregate View and Edit for Trigger CRD ([#790](https://github.com/tektoncd/triggers/pull/790))
+
+* Remove explicit pinning of golang version ([#794](https://github.com/tektoncd/triggers/pull/794))
+
+* Fix TestTrigger_Error using deterministic inputs ([#798](https://github.com/tektoncd/triggers/pull/798))
```
Comment (savitaashture) [inline openshift/release/release-notes.md:294]: this also

File: openshift/release/release-notes.md (hunk 3 above)
Comment (vdemeester) [reply openshift/release/release-notes.md:141]: @savitaashture before doing that, is there any of these fix fixing a bug that was reported / mentionned by customers (jira, …) ?

Comment (savitaashture) [reply openshift/release/release-notes.md:141]: no 
There are no such bugs

**Review by vdemeester — APPROVED** [review]

File: openshift/release/release-notes.md [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+## Fixes :bug:
+* Merge extensions into body for webhook interceptors ([#860](https://github.com/tektoncd/triggers/pull/860))
+
+  Extensions added by a CEL Interceptor will be passed on to webhook interceptors by merging the extension fields into the event body under a `extensions` field. 
+  See docs/eventlisteners.md ##chaining-interceptors for more details.
+
+# Tekton Triggers v0.10.1
+
+## Fixes :bug:
+* Fix example SA permissions ([#848](https://github.com/tektoncd/triggers/pull/848))
```
Comment (khrm) [inline openshift/release/release-notes.md:12]: This can also be removed.

File: openshift/release/release-notes.md [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+  Previously, parameters were escaped as they were being replaced into a TriggerTemplate, by replacing double-quotes " with an escaped version ", this functionality has been removed, as it was breaking quoted strings, and in some cases, rendering the resulting output unparseable.
+
+  action required: If you were relying on the escaping, you can retain the old behaviour by adding an annotation to an affected TriggerTemplate, `triggers.tekton.dev/old-escape-quotes: "true"`
+
+# Tekton Triggers v0.9.1
+
+## Fixes :bug:
+* Revert "dibyo's point about only EL sink/interceptors reading secrets" ([#804](https://github.com/tektoncd/triggers/pull/804))
+  Allow webhook to create secrets (for TLS certs)
```
Comment (khrm) [inline openshift/release/release-notes.md:117]: This one is also not needed I think.

File: openshift/release/release-notes.md [hunk 11]
```diff
@@ -0,0 +1,219 @@
...
+  The template.name field is deprecated (in favor of template.ref) and will be removed in a future release.
+
+* Adds support for name/value embedded Bindings ([#768](https://github.com/tektoncd/triggers/pull/768))
+
+  The old syntax for embedded TriggerBindings (using `spec.params`) is deprecated. Use the new name/value fields instead:  
+  
+## Backwards incompatible changes :rotating_light:
+In current release:
+
+* Bump Knative/K8s dependencies. ([#784](https://github.com/tektoncd/triggers/pull/784))
```
Comment (khrm) [inline openshift/release/release-notes.md:204]: Not needed.

**Comment by savitaashture** [comment]
@khrm addressed review comments

**Review by khrm — APPROVED** [review]
/lgtm

**Pull request merged**